app/assets/stylesheets/header.scss

```diff
@@ -13,35 +21,36 @@
   margin-right: 50px;
 }
 .top-nav ul{
-  padding:0;
-  margin:0;
+  padding: 0;
+  margin: 0;
 }
 .top-nav ul li{
-  display:inline-block;
+  display: inline-block;
   margin: .5em 2em 0;
 }
 .top-nav ul li a{
-  color:#97999B;
-  font-size:17px;
-  transition:.5s all;
-  -webkit-transition:.5s all;
-  -moz-transition:.5s all;
-  -o-transition:.5s all;
-  -ms-transition:.5s all;
+  color: #686c6d;
+  font-size: 17px;
+  line-height: 40px;
+  transition: .5s all;
+  -webkit-transition: .5s all;
+  -moz-transition: .5s all;
+  -o-transition: .5s all;
+  -ms-transition: .5s all;
 }
 .top-nav ul li a:hover{
   color: #5DC9C9;
-  text-decoration:none;
+  text-decoration: none;
   padding: 1.4em 0 0;
   border-top: 3px solid #5DC9C9;
 }
 .head-logo a{
-  display:block;
+  display: block;
 }
 .top-nav p {
   margin: 2.1em 0;
 }
 .top-nav p a:hover{
-  color:#5DC9C9;
-  text-decoration:none;
+  color: #5DC9C9;
```

Comment:
Color literals like `#5DC9C9` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color `#5DC9C9` should be written as `#5dc9c9`